pages/common/boot.md

```diff
@@ -0,0 +1,23 @@
+# boot
+
+> Build tooling for Clojure.
+
+- Generate scaffolding for a new project based on a template:
+
+`boot -d boot/new new --template {{template_name}} --name {{project_name}}`
```

Comment:
Is there an expanded form of `-d` ? Because the other options use the expanded form.